src/libtomahawk-playdarapi/Api_v1_5.cpp

```diff
@@ -105,7 +105,7 @@ Api_v1_5::playback( QxtWebRequestEvent* event, const QString& command )
             } else {
                 trackInfo.insert( "resolvedBy", "<unknown resolver>" );
             }
-            trackInfo.insert( "score", currentTrack->score() );
+//FIXME?            trackInfo.insert( "score", currentTrack->score() );
```

Comment:
Yeah, we'd need some predefined levels for things like "basically works all the time", "should work most of the time", "rarely ever works" etc.. then it might become useful.. or maybe this is just overengeneering ... and we shouldn't care about it now.